flink-table/flink-table-common/src/main/java/org/apache/flink/table/annotation/FunctionHint.java

```diff
@@ -175,13 +178,40 @@
     ArgumentHint[] arguments() default {};
 
     /**
-     * Explicitly defines the intermediate result type that a function uses as accumulator.
+     * Explicitly defines the intermediate result type (i.e. state entry) that an aggregating
+     * function uses as its accumulator. The entry is managed by the framework (usually via Flink's
+     * managed state).
```

Comment:
it would be worth saying how this is managed by the frame =work if not using managed state.